google-oauth-client/src/main/java/com/google/api/client/auth/oauth/OAuthHmacSha256Signer.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2010 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.api.client.auth.oauth;
+
+import com.google.api.client.util.Base64;
+import com.google.api.client.util.Beta;
+import com.google.api.client.util.StringUtils;
+import java.security.GeneralSecurityException;
+import javax.crypto.Mac;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+
+/**
+ * {@link Beta} <br>
+ * OAuth {@code "HMAC-SHA256"} signature method.
+ */
+@Beta
```

Comment:
auth leader (not me) needs to figure out if we want this and if it should be beta

Reply:
Since the interface the `OAuthSigner` interface itself is Beta I took that as a signal that any classes implementing it should also be Beta.

Reply:
Either way annotation might be enough. Is there a reason to include this in doc comment too?

Reply:
removed this from the doc comment